go/vt/vttablet/onlineddl/schema.go

```diff
@@ -359,6 +359,10 @@ const (
 			AND cleanup_timestamp IS NULL
 			AND completed_timestamp IS NULL
 	`
+	sqlShowMigrationsWhere = `SELECT *
+		FROM _vt.schema_migrations
+		%s
+	`
```

Comment:
We can no longer use a hardcoded database name of `_vt` in vttablet. You'll have to use the result of `sidecardb.GetIdentifier()`.

Reply:
This means all the queries in this file needs to be changed.

Reply:
Ah, never mind. I’m sorry! I’m was forgetting that for OnlineDDL we do the replacement if needed. So this is fine.

Reply:
That’s what this is for: https://github.com/vitessio/vitess/blob/c4be8e297bed6ff08ba760322da64061652d06ff/go/vt/vttablet/onlineddl/executor.go#L198

So I’ll resolve this thread and approve. Sorry again.